multimodal/src/autogluon/multimodal/optimization/RKDLoss.py

```diff
@@ -0,0 +1,70 @@
+# Reference: https://github.com/lenscloth/RKD/blob/master/metric/loss.py
```

Comment:
Consider to change the file name to be rkd_loss.py . Avoid using capitalized characters in file names.

Reply:
Done